### PR TITLE
AG-10614 Move minichart to enterprise module

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-10614

### CODE REVIEW

File: packages/ag-charts-community/src/chart/chart.ts [hunk 1]
```diff
@@ -1435,10 +1435,13 @@ export abstract class Chart extends Observable implements AgChartInstance {
             this.zoomManager.updateZoom();
         }
 
-        const miniChartInstance = navigatorModule?.miniChartInstance;
-        if (miniChartInstance != null) {
-            const seriesStatus = this.applySeries(miniChartInstance, processedOptions);
-            this.applyAxes(miniChartInstance, processedOptions, seriesStatus);
+        const navigator = this.modules.get('navigator') as any;
```
Comment (iMoses) [inline packages/ag-charts-community/src/chart/chart.ts]: notice you already have the `navigatorModule` 8 lines up.

Comment (jacobp100) [reply packages/ag-charts-community/src/chart/chart.ts]: Updated

File: packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const EXPECTED_ENTERPRISE_MODULES: EnterpriseModuleStub[] = [
         chartTypes: ['cartesian', 'polar', 'hierarchy'],
         identifier: 'gradient',
     },
+    { type: 'root', optionsKey: 'miniChart', chartTypes: ['cartesian'], identifier: 'mini-chart' },
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts]: Nit: Drop `identifier`, it's mostly for series and axes cases where a `type` discriminator is involved (see other `'root'` cases above where we don't specify this too).

Comment (alantreadway) [inline packages/ag-charts-community/src/chart/factory/expectedEnterpriseModules.ts]: I'm debating if we should add a `type: 'navigator'` case, as this isn't technically an (options) root module - could we do that relatively easily?

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**